test/parallel/test-buffer.js

```diff
@@ -749,6 +749,11 @@ for (let i = 0; i < 256; i++) {
   assert.equal(hexb2[i], hexb[i]);
 }
 
+//#6770 Test single hex character throws TypeError
+assert.throws(function() {
+  Buffer.from('A', 'hex');
+}, TypeError);
```

Comment:
Sure! Is there a reason why hex throws an error for odd lengths but base64 doesn't?